File: frontweb/Data/Entities/Cliente.cs
using System.Collections.Generic;

namespace PetWorldWeb.Data.Entities
{
    public class Cliente
    {
        public long Id { get; set; }
        public string Nome { get; set; }
        public string Email { get; set; }
        public string Endereco { get; set; }
        public string Telefone { get; set; }

        public List<Pet> Pets { get; set; }
    }
}

File: frontend/src/types/Cliente.ts
import Pet from "./Pet"

export type Cliente = {
    id: number
    nome: string
    email: string
    endereco: string
    telefone: string
    pets?: Pet[]
}

export type ClienteInsert = {
    nome: string
    email: string
    endereco: string
    telefone: string
}

export default Cliente;